<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Registros</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="background_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 222, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QTableWidget" name="dataTable">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>255</y>
       <width>1200</width>
       <height>400</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1200</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1200</width>
       <height>500</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>9</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>110</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>110</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Secadora</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Temperatura</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Temperatura2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Humedad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Operador</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Variedad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Batch</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="titleLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Arial Narrow&quot;;
color: rgb(148, 148, 148);</string>
     </property>
     <property name="text">
      <string>Registros secadoras</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="datesLayout">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="horizontalSpacing">
       <number>19</number>
      </property>
      <property name="verticalSpacing">
       <number>11</number>
      </property>
      <item row="0" column="1">
       <widget class="QDateEdit" name="beginDateEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial Narrow&quot;;</string>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2021</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2021</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelBeginDate">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Arial Narrow&quot;;
</string>
        </property>
        <property name="text">
         <string>Fecha inicio</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEndDate">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Arial Narrow&quot;;</string>
        </property>
        <property name="text">
         <string>Fecha fin</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="endDateEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial Narrow&quot;;</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2021</year>
          <month>12</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="selectFilterLayout">
      <item>
       <widget class="QComboBox" name="columnFilterBox">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial Narrow&quot;;</string>
        </property>
        <property name="currentText">
         <string>Seleccione una columna</string>
        </property>
        <item>
         <property name="text">
          <string>Seleccione una columna</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Secadora</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Temperatura</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Temperatura2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Humedad</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Operador</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Variedad</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Batch</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial Narrow&quot;;</string>
        </property>
        <property name="placeholderText">
         <string>Ingrese el valor a buscar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="lineTop">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>1011</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line2Top">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="lineLeft">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>20</width>
       <height>641</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="lineRight">
     <property name="geometry">
      <rect>
       <x>1251</x>
       <y>30</y>
       <width>20</width>
       <height>641</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="lineBottom">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>661</y>
       <width>1233</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelFilterByDates">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Arial Narrow&quot;;</string>
     </property>
     <property name="text">
      <string>Filtro por fecha</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>80</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="chartDateLayout">
      <property name="verticalSpacing">
       <number>25</number>
      </property>
      <item row="0" column="1">
       <widget class="QDateEdit" name="chartDate">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial Narrow&quot;;</string>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2021</year>
          <month>12</month>
          <day>10</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2021</year>
          <month>12</month>
          <day>10</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelChartDate">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Arial Narrow&quot;;
</string>
        </property>
        <property name="text">
         <string>Fecha</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="labelChart">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>50</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Arial Narrow&quot;;</string>
     </property>
     <property name="text">
      <string>Gráfico de tendencia</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>120</y>
       <width>181</width>
       <height>88</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="chartSecadoraLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="drawChartButton">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Arial Narrow&quot;;
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Dibujar gráfico</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="chartComboBox">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial Narrow&quot;;</string>
        </property>
        <property name="currentText">
         <string>Secadora</string>
        </property>
        <item>
         <property name="text">
          <string>Secadora</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AGRIMAQ 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KW 325</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FARM 7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FARM 9</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="logoLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>40</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>120</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../resources/logomsi.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelFilterByColumn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Arial Narrow&quot;;</string>
     </property>
     <property name="text">
      <string>Filtro por columna</string>
     </property>
    </widget>
    <zorder>dataTable</zorder>
    <zorder>formLayoutWidget</zorder>
    <zorder>verticalLayoutWidget</zorder>
    <zorder>lineTop</zorder>
    <zorder>line2Top</zorder>
    <zorder>lineLeft</zorder>
    <zorder>lineRight</zorder>
    <zorder>lineBottom</zorder>
    <zorder>titleLabel</zorder>
    <zorder>labelFilterByDates</zorder>
    <zorder>gridLayoutWidget</zorder>
    <zorder>labelChart</zorder>
    <zorder>gridLayoutWidget_2</zorder>
    <zorder>logoLabel</zorder>
    <zorder>labelFilterByColumn</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources/filter.qrc"/>
 </resources>
 <connections/>
</ui>
